examples/AuthServer.js

```diff
@@ -0,0 +1,51 @@
+const express = require('express');
+const uport = require('../lib/index.js')
+const bodyParser = require('body-parser')
+
+const app = express()
+app.use(bodyParser.json({ type: '*/*' }))
+const signer = uport.SimpleSigner('955d253bca62bf380e34a36ed3802895e4adc297fa861e22bb7ee3e66714592b')
+
+const authCredentials = new uport.AuthCredentials({
+  appName: 'Credential Tutorial',
+  address: '0x15bcc15904f2317365cdb6beaf473b2315405763',
+  signer: signer
+})
+
+app.get('/authrequest', (req, res) => {
+    // Browser requests to start authentication and receives a request token in response
+    credentials.createRequest({
+      request: ['name'],
+      // A reachable ip address
+      callbackUrl: 'http://192.168.1.101:8081/authResponse'
+    }).then( (requestToken) => {
+      const uri = 'me.uport:me?requestToken=' + requestToken
+      const.log(uri)
+      res.send(uri);
+    })
+})
+
+app.post('/authresponse', (req, res) => {
+  // Mobile app posts response to challenge here
+  const jwt = req.body.access_token
+  const response = 'Successful Authentication'
+  receive(jwt, response).then( credentials => {
+    console.log(response)
+  }).catch(err => {
+    console.log('Authentication Failed')
+  })
+})
+
+app.get('/authresponse/:pairID', (req, res) => {
```

Comment:
@pelle actually on second thought it may make more sense to send all responses back to the server which are responses for requests generated on the same server. There are a few benefits, 1) it would always allow the flow to be the same from the perspective of the server (instead of sometimes the server has to write the response to storage, other times it has to immediately return the response, its not clear how to differentiate these), 2) it would allow the flow to be more easily managed in the library and wrapped up for end users, it could always return the server response  as a result of valid/invalid CR (the alternative would be, when in mobile browser the lib would get the response from mobile of the challenge (then have to get the server response), while on desktop the lib would get the server response (not the mobile response, alternatively getting the mobile response would not make much sense here, as it would have been sent to the server then returned here by polling, then sent back to the server)) 3) may allow us to wrap up certain best practices in the libraries, like have a certain amount of control over how the response to the browser is paired to the challenge/response. Ultimately either way a developer can do as they please, but this would likely improve the experience in the libraries.